src/cascadia/TerminalApp/CascadiaSettings.cpp

```diff
@@ -215,7 +215,7 @@ void CascadiaSettings::_CreateDefaultProfiles()
     if (_isPowerShellCoreInstalled(psCoreCmdline))
     {
         auto pwshProfile{ _CreateDefaultProfile(L"PowerShell Core") };
-        pwshProfile.SetCommandline(psCoreCmdline);
+        pwshProfile.SetCommandline(std::move(psCoreCmdline));

```

Comment:
Precisely.

Reply:
BTW, this effectively means that if the creation of the string raises an exception, it will happen at the call site, in `CascadiaSettings::_CreateDefaultProfiles()`, not in `Profile::SetCommandLine()` (which is rightfully `noexcept`).